valid date program in c++

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+
```

Comment:
make a function ISValidDate(int year,int month,int day) instead of doing everything in main

Reply:
Sure.